src/librustc_mir/build/expr/as_temp.rs

```diff
@@ -6,7 +6,7 @@ use crate::hair::*;
 use rustc::middle::region;
 use rustc::mir::*;
 
-impl<'a, 'gcx, 'tcx> Builder<'a, 'gcx, 'tcx> {
+impl<'a, 'tcx> Builder<'a, 'tcx> {
```

Comment:
You can replace headers like these with `impl Builder<'_, 'tcx>` if you want.

Reply:
I feel like this is a job for rustfmt, not me =P

Reply:
Not `rustfmt` but maybe `rustfix`. Don't know if we have a lint for it yet though.